src/resource/resource_base.cpp

```diff
@@ -33,5 +33,15 @@ ResourceType ResourceBase::type() {
     return {"resource"};
 }
 
+ResourceName ResourceBase::get_resource_name(std::string name) {
+    ResourceName r;
+    *r.mutable_namespace_() = RDK;
+    *r.mutable_type() = "resource";
```

Comment:
[nit] add `const std::string RESOURCE = "resource";` to `src/common/utils.hpp`.

Reply:
Yeah good catch, updated :)